querydsl-sql-codegen/src/main/java/com/mysema/query/sql/codegen/MetaDataExporter.java

```diff
@@ -109,6 +109,9 @@
     private boolean exportForeignKeys = true;
 
     private boolean spatial = false;
+    
+    @Nullable
+    private String tableTypesToExport; 
```

Comment:
The property comes in as a comma-separated list from the pom:

`<tableTypesToExport>TABLE, VIEW</tableTypesToExport>`

and AbstractMetaDataExportMojo just calls the setter.  If we change it to a List<String> here, we would have to tokenize the string in that class instead.  I basically replicated what was done for tableNamePattern, which also allows a comma separated string.